index.js

```diff
@@ -113,8 +116,15 @@ async function findInRepoFiles(cwd) {
   return globby(joinPaths(inRepoFolders, ['**/*.js', '**/*.hbs', '**/*.emblem']), { cwd });
 }
 
-async function findTranslationFiles(cwd) {
-  let inputFolders = ['', ...findInRepoPaths(cwd)];
+async function findOwnTranslationFiles(cwd) {
+  return findTranslationFiles(cwd, ['', ...findInRepoPaths(cwd)]);
+}
+
+async function findAddonTranslationFiles(cwd) {
+  return findTranslationFiles(cwd, ['node_modules/*', 'node_modules/@*/*']);
```

Comment:
Hmm just a thought but maybe we could search the only the packages that are in the package.json, this might help make it a little more performant

Reply:
Yeah, I was also not sure if this is best (although it is still performant in our codebase, which contains plenty of dependencies). Maybe, instead of looking up in the package.json, I can add an entry to the config, like `externalPaths`, where you can specify an array like `externalPaths: ['some-dependency', '@company/scoped-dependency']`, or even allowing a regex like `['@company/*']` (which would be something that would work for us)?

Reply:
Yeah, it is probably fine, all i was thinking is it is likely you would only use translations for 1 or two packages and seems a lot to check them all. i think the external paths looks like a good bet, having to specify isn't ideal but we can add a section to the readme to make it clear what to do for addon translations. Also i haven't mentioned it but thanks for the work!

Reply:
Yeah, I agree that using translations from many other packages is not very likely indeed, most apps probably don't want to depend too much on external translations, so defining it explicitly in the config makes a bit more sense then I think. It's also more flexible in case an addon doesn't define translations in the root or something, so I'll go for that option then :+1:.

> Also i haven't mentioned it but thanks for the work!

Thanks :slightly_smiling_face: !

Reply:
@Mikek2252 I've added the config option + a section in the readme, let me know what you think :+1: